ex01.exs

```diff
@@ -26,8 +26,23 @@ defmodule Ex01 do
         2 is the program well laid out,  appropriately using indentation,
           blank lines, vertical alignment
   """
-  
+
   def counter(value \\ 0) do
+    receive do
+      {:next, from} ->
+        send(from, {:next_is, value})
+    end
```

Comment:
No penalty, but my old, tired eyes find

~~~
counter value + 1
~~~

harder to parse than

~~~
counter(value + 1)
~~~